### DISCUSSION TITLE
8.2.2023 huoltoikkuna

### DISCUSSION THREAD

**Discussion by CodoDerDritte** [KohaSuomi/Koha#406] [Huolto- ja häiriötiedotteet]
Keskiviikon huoltoikkunassa vaihdetaan palomuurilaitteiden muistikammat kummallekkin palomuurilaitteelle.

Varsinaista käyttökatkoa tästä ei pitäisi aiheutua, mutta tunneloitujen automaattien heräilyssä voi olla viivettä. Liikenne siirretään keskiviikkona noin 7.30 - 7.45 välisenä aikana kulkemaan kakkospalomuurin kautta ja voi olla, ettei tunneleita ehditä ennen kello yhdeksää neuvotella uusiksi, jolloin tunneloidut automaatit putoavat kyydistä. Tämä koskee Hellen, Vaaran ja Vaskin tunneloituja automaatteja. Muut näiden kimppojen automaatit sekä muiden kimppojen automaatit toimivat normaalisti.

Olisi hyvä jos tunnelia kirjaston päässä hoitava taho voisi käydä tarvittaessa painamassa connect (tai reconnect) nappulaa, jotta yhteyden uudelleenneuvottelu käynnistyisi liikenteen siirtämisen jälkeen. Tunneleiden pitäisi siirtyä ilman nappulan painamistakin, mutta riippuen kirjaston tunnelipään konfiguraatiosta niin ei välttämättä tapahdu tai siihen voi kulua turhan pitkä aika.

Liikenne siirretään takaisin ykkösmuurille vasta kirjastojen sulkemisen jälkeen, joten siitä vastaavaa häiriötä ei pitäisi aiheutua.

**Comment by CodoDerDritte** [comment]
Huomenta! Aloitellaan kuukausihuollot tuota pikaa. Ensin kakkospalomuuri menee alas ja siihen vaihdetaan muistikammat. Tämä ei näy kirjastoissa mitenkään.

**Comment by CodoDerDritte** [comment]
Kakkosmuuri on huollettu ja siirrämme liikenteen sille. Tässä on siis se paikka kun tunneloidut automaatit voivat pudota. Niiden kanssa kannattaa odotella hetki, että josko tunneli aukeaisi itsekseen uudelleen. Jos ei, niin sitten yhteys tunnelia hallinnoivaan tahoon ja uudelleenyhdistäminen voi olla tarpeen.

**Comment by CodoDerDritte** [comment]
Kaksi kolmesta tunnelista on nyt ylhäällä, eli yhtä vielä odotellaan. Ikävä kyllä ei ole tarkalleen tiedossa mikä niistä ei ole vielä noussut. Ykkösmuuri ajetaan nyt joka tapauksessa alas huoltoa varten. Tämäkään ei näy kirjastoissa.

**Comment by CodoDerDritte** [comment]
Liikenne tosiaan siirretään takaisin ykkösmuurille vasta illalla kirjastojen sulkemisen jälkeen, joten potentiaalisesti kirjastojen toimintaan vaikuttavat huoltotoimenpiteet olivat tämän kuun osalta tässä. Ykkösmuuri on työn alla ja ilmoitan vielä kun saamme siihenkin kammat vaihdettua ja laitteen takaisin käyntiin.

**Comment by CodoDerDritte** [comment]
Viimeinenkin tunneli on nyt taas auki. Ykköspalomuurin muistit vaihdettu ja laitetta käynnistellään parhaillaan.

**Comment by pknuuti** [comment]
Kohan ylläpitoon ei nyt pääse, tulee ilmoitus 404 not found. Näin ainakin Vaarassa.

**Reply by CodoDerDritte** [reply]
Siis administrationiin? Minulla ei tule täällä vaan toimii ihan normaalisti?

**Reply by pknuuti** [reply]
Juu. Nyt ei toimi ollenkaan eli ei tapahdu mitään, ei edes virheilmoitusta.

**Reply by pknuuti** [reply]
Eipäs kun toimii nyt.

**Reply by CodoDerDritte** [reply]
Olipa outo härö. En kyllä tiedä äkkipäätä mistä tuo olisi voinut johtua :D

**Reply by CodoDerDritte** [reply]
Täältä katsottuna toimi tosiaan kyllä koko ajan ihan normaalisti, otappa tuostakin nyt selvä sitten.

**Reply by pknuuti** [reply]
Ei voi aina ymmärtää. No, onneksi oli kovin tilapäinen häiriö. Piti vain tarkistaa yksi asetus.

**Comment by CodoDerDritte** [comment]
Ykkösmuuri imaisikin ilmeisesti pyytämättä ja yllättäen ne tunnelit takaisin, eli ne siirtyivät sitten jo takaisin sinne.

**Comment by CodoDerDritte** [comment]
Tämän kuukauden huolto on nyt valmis. Ensi kuussa sitten uudet kujeet.

**Comment by CodoDerDritte** [comment]
Jaahas, jotain hämärää on vielä niissä tunneleissa. Esimerkiksi Hellessä näkyy operaattorin mukaan 88 verkkoa(!) Selvitellään.

**Comment by CodoDerDritte** [comment]
Vaara ja Vaski on ok, niissä homma näyttää siltä miltä pitääkin. Hellessä siis on joku solmu nyt.

**Comment by CodoDerDritte** [comment]
Helle-episodi jatkuu, tunneli käyttäytyy nyt tosi tuhmasti. Jostain syystä uusia yhteyksiä muodostuu nyt koko ajan, niitä on jo neljättä sataa(!)

**Reply by CodoDerDritte** [reply]
Yhteyksien jatkuva kumuloituminen aiheuttaa erittäin todennäköisesti ongelmia jossain vaiheessa. Silloin paitsi Hellen, niin myös kaikkien muiden kimppojen pääsy Kohaan estyy. Tästä syystä jouduimme katkaisemaan Porvoon tunneliyhteyden. Tunneloitu palautusautomaatti on siis nyt kokonaan pois linjalta. 

Ongelman selvittämiseksi ja korjaamiseksi tarvitaan yhteys tunnelin ylläpitäjään Porvoon päässä.

**Reply by CodoDerDritte** [reply]
Konesalioperaattori ja Porvoon palomuurin ylläpidosta vastaava taho selvittävät ongelmaa.

**Reply by CodoDerDritte** [reply]
Operaattori ja Porvoon it-väki eivät löytäneet eilen syytä kummitteluun, selvitykset jatkuvat tänään. Mukana selvityksissä on myös Telian väkeä.
